src/Cache/Internal/ScsDataClient.php

```diff
@@ -206,13 +206,13 @@ public function set(string $cacheName, string $key, string $value, int $ttlSecon
             $call = $this->grpcManager->client->Set(
                 $setRequest, ["cache" => [$cacheName]], ["timeout" => $this->timeout]
             );
-            $response = $this->processCall($call);
+            $this->processCall($call);
         } catch (SdkError $e) {
             return new SetError($e);
         } catch (Exception $e) {
             return new SetError(new UnknownError($e->getMessage()));
         }
-        return new SetSuccess($response, $key, $value);
+        return new SetSuccess();
```

Comment:
Sorry if I missed in your previous PRs, but we don't take any params to create an instance of the `SetSuccess` anymore?

Reply:
No, we don't.